client/webpack-hot-middleware-client.js

```diff
@@ -9,8 +9,8 @@ const {
 
 export default () => {
   webpackHotMiddlewareClient.setOptionsAndConnect({
-    overlay: false,
```

Comment:
Yeah had to check the source of webpack-hot-middleware, it's interesting because this worked before. Maybe their option parsing was wrong and they fixed it.